python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -107,6 +111,12 @@ def __init__(self, config_dir=None, initial_state=State.STANDBY):
                                               "reportedFilterPosition",
                                               "reportedDisperserPosition"])
 
+        # self.atcs = ATCS(intended_usage=7)  # this is OffsettingForATAOS Usage
+
+        self.log.debug('loading ATCS from ataos_csc')
+        # self.atcs = ATCS(domain=self.domain)
+        self.atcs = ATCS(domain=self.domain, intended_usage=ATCSUsages.OffsettingForATAOS)
```

Comment:
So, I think this would require slightly larger refectory of the code. Because the ATCS defines Remotes for components we are also defining here. So maybe move to using those remotes instead?

Reply:
Partially implemented. Due to a callback being used in the ATAOS for telescope position, the aget on that same position inside the ATCS causes failures. I worked around this by using the ATCS remotes for the hexapod, pneumatics, and pointing, but had to duplicate a remote for the atmcs.

Reply:
This is now fixed and all remotes use ATCS remotes.